lib/idx/remediate.ts

```diff
@@ -211,11 +197,55 @@ export async function remediate(
         }
         return remediate(idxResponse, valuesWithoutExecutedAction, options); // recursive call
       }
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      else if (typeof action ==='string' && action in remediators!) {
+        const remediation = neededToProceed.find(({ name }) => name === action);
+        if (!remediation) {
+          continue;
+        }
+        foundRemediationCandidate = true;   // signal remediation(s) were passed as actions
```

Comment:
here action means remediation that does not need any data from user interaction, how about return idx.proceed(name) directly?